cmd/opampsupervisor/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.24-alpine3.20
```

Comment:
wouldn't it be possible to use an image with a smaller footprint? maybe just alpine? i don't think we really need golang inside the image
a scratch image would also be possible but then using it as a base image is gonna be a hassle

Reply:
Updated to use `alpine:3.20`